react/week2/my-react-app/src/App.js

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from "react";
+import Todos from './Todos/Todos';
+import AddTodo from './Todos/AddTodo';
+import './App.css';
+
+const arrayOfTodos = [
+  {
+    id: 1,
+    description: "Get out of bed",
+  },
+  {
+    id: 2,
+    description: "Brush teeth",
+  },
+  {
+    id: 3,
+    description: "Eat breakfast",
+  },
+];
+
+export const Timer = ({maxRange})=>{
+  const [counter,setCounter]= useState(maxRange)
+  useEffect(()=>{
+      if(counter){
+         setTimeout(() =>setCounter(counter + 1),1000)
+      }
+  })
+  return (
+      <div>You are using this webpage from {counter} seconds</div>
+  )
+}
+
+function App() {
+  const [todos, setTodos] = useState(arrayOfTodos);
+ 
+  //adding todo
+  const addTodo = (description) => {
+    if (description) {
+      const id = Math.floor(Math.random() * (100 - 4 + 1)) + 4;
+      console.log(id)
+      setTodos(todos.concat({
```

Comment:
if you are using setTodos to add a new item you can simply destructure the array and add the new todo 
```suggestion
      setTodos(...todos, {
```